src/lib/media/manip.ts

```diff
@@ -114,8 +114,39 @@ export async function saveImageToMediaLibrary({uri}: {uri: string}) {
       // as the starting image, or put it directly into the album
       const album = await MediaLibrary.getAlbumAsync(ALBUM_NAME)
       if (album) {
-        // if album exists, put the image straight in there
-        await MediaLibrary.createAssetAsync(imagePath, album)
+        // try and migrate if needed
+        try {
+          if (await MediaLibrary.albumNeedsMigrationAsync(album)) {
+            await MediaLibrary.migrateAlbumIfNeededAsync(album)
+          }
+        } catch (err) {
+          logger.debug('Attempted and failed to migrate album', {
```

Comment:
Do we not care to get these into Sentry? `info` would add a breadcrumb at least if that's helpful